conans/client/userio.py

```diff
@@ -26,14 +27,17 @@ def color_enabled(stream):
 
 def init_colorama(stream):
     import colorama
-
     if not color_enabled(stream):
-        if os.getenv("NO_COLOR") != "1":
-            colorama.init(strip=True)
-        return False
+        if os.getenv("NO_COLOR") == "1":
+            # Do not initialize colorama
```

Comment:
Before anyone says this can be simplified. I know, but honestly I prefer it like this to show that the "NO_COLOR" means no initialize colorama, otherwise initialize colorama but striping.